frontend/src/components/VHeader/VHeaderMobile/VContentSettingsModalContent.vue

```diff
@@ -49,11 +49,17 @@
           @select="$emit('select', $event)"
         />
       </VTabPanel>
-      <VTabPanel v-if="showFilters" id="filters">
+      <!-- Horizontal padding removed to display divider. -->
+      <VTabPanel v-if="showFilters" id="filters" class="px-0">
         <VSearchGridFilter
+          class="px-6"
           :show-filter-header="false"
           :change-tab-order="false"
         />
+        <VSafeBrowsing
+          v-if="isSensitiveContentEnabled"
+          class="border-t border-dark-charcoal-20 px-6 pt-6"
```

Comment:
Nice to add these styles in their sidebar, instead of hardcoding them into the component 👍